pkgs/development/python-modules/pyttsx3/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, python3Packages, buildPythonPackage, espeak-ng, ffmpeg}:
```

Comment:
Unlike top-level packages, Python packages in `pkgs/development/python-modules` should get dependencies from the `python3.pkgs` scope as arguments directly – that means do not pass `python3Packages` here.